code/src/index.css

```diff
@@ -1,13 +1,137 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+
+
 body {
   margin: 0;
+  width: 100vw;
+  box-sizing: border-box;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  overflow-x: hidden;
+ 
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.happyform{
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: auto;
+ padding: 10px;
+  border: 1px black solid;
+  padding-top: 3px;
+  background-color: #F2F0F0;
+  box-shadow: 5px 7px 0px 0px black;
+  width: 360px;
+  margin-top: 20px;
+}
+
+.newThought{
+  grid-column: 1/2;
+  justify-self: center;
+}
+.loveBtn{
+  grid-column: 1/2;
+  justify-self: center;
+  margin-top: 7px;
+  font-size: 11px;
+  border-radius: 20px;
+  border: 0;
+  padding: 7px;
+  padding-left: 15px;
+  padding-right: 15px;
+  background-color: #ffadd6;
+  letter-spacing: 0.5px;
+}
+
+.loveBtn:hover {
+  cursor: pointer;
+  box-shadow: 0 0 10px rgba(254, 53, 133, 0.839);;
+  /* box-shadow: 0 0 10px rgba(255, 173, 173, 0.634); */
+}
+
+
+.loveBtn:active {
+  background: rgba(253, 149, 190, 0.839);
+}
+
+/* msg*/
+
+  .thoughts{
+    display: grid;
+    justify-content: center;
+    justify-self: center;
+    border: solid 2px rgb(226, 225, 225);
+    justify-content: space-between;
+    width: 50vw;
+    margin: 15px auto;
+    padding: 0 20px;
+    height: 130px;
+    background-color: white;
+    box-shadow: 5px 7px 0px 0px black;
+    margin-bottom: 30px;
+    overflow-wrap: break-word;
+    width: 335px;
+  }
+
+  .thoughts-p{
+    margin: 20px;
+    padding-top: 10pxpx;
+    width: 100%;
+
+  }
+.buttoncontainer{
+  display: flex;
+}
+.giveLove{
+  display: flex;
+justify-content: flex-end;
+  width: 335px;
+}
+.likes {
+ margin: 5px;
+}
+.time{
+ margin-bottom: 5px;
+  opacity: .7;
+    position: relative;
+    display: flex;
+  justify-content: flex-end;
```

Comment:
If a message is too long (because someone has written without spaces) the time moves out in the perifery. But if you change to "anywhere" in the above comment, you get past that without changing the alignment.